components/Profile/TeamCard.js

```diff
@@ -9,6 +9,7 @@ export default function TeamCard({ team }) {
                 <Div>
                     <Image
                         src={team.hackathon.image}
+                        alt="team image"
```

Comment:
I would like it not be to be this brief. It can be something like ' {name of hackathon} hackathon ' Coz it's actually the image of the hackathon, not team which I guess is a bit confusing considering that we get it from team object 😅